docs/source/upgrade-guide.rst

```diff
@@ -4,6 +4,134 @@ Upgrade Guide
 This upgrade guide lists all breaking changes in plone.restapi and explains the necessary steps that are needed to upgrade to the lastest version.
 
 
+Upgrading to plone.restapi 4.x
+------------------------------
+
+Vocabularies
+^^^^^^^^^^^^
+
+Choice fields using named vocabularies are now serialized
```

Comment:
In fact, all Choices and Lists fields that contains a vocabulary are affected by the change.